daemon/cluster/executor/container/health_test.go

```diff
@@ -0,0 +1,102 @@
+// +build !windows
```

Comment:
No. I tested in Linux, It passed. After adding this test, there was one time that all tests are passed, except on windows.

Also I notice that both `docker_cli_swarm_test` and `docker_cli_service_update_test` are built !windows